pom.xml

```diff
@@ -60,6 +60,7 @@
       <hibernate-validator-version>8.0.0.Final</hibernate-validator-version>
       <io-admin-shell-aas-model>1.2.0</io-admin-shell-aas-model>
       <io-admin-shell-aas-serializer>1.2.1</io-admin-shell-aas-serializer>
+      <aas4j-version>1.0.0-milestone-03.1</aas4j-version>
```

Comment:
If this is the version for a library that should be newly added (i.e., a new dependency), please add the corresponding `<dependency>` block(s) in the `<dependencyManagement>` section below.
For components that use the parent pom, they will then refer to this dependency just like this:
```xml
<dependency>
  <groupId>foo.bar</groupId>
  <artifactId>baz</artifactId>
</dependency>
```
i.e., without an explicit `<version>`.

Reply:
thanks for pointing out. You are completely right. It slipped through when I reverted the old commit to adjust the version property for aas4j. Fixed now